StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -5,8 +5,44 @@
 
 import Foundation
 
-/// An `Event` object represent an event in the chat system.
+/// An `Event` object representing an event in the chat system.
 public protocol Event {
     /// The underlying raw type of the incoming string.
     static var eventRawType: String { get }
 }
+
+/// The DTO object mirroring the JSON representation of an event.
+struct EventPayload<ExtraData: ExtraDataTypes>: Decodable {
+    let eventType: String
+    
+    let connectionId: String?
```

Comment:
The opposite situation is here. The `connectionId` we only need for the `healthCheck` event.

Reply:
This is just the payload so it reflects the JSON structure we can receive from WS. All these values are optional. We have the distinction you're mentioning later for events themselves. For example, we have:
```swift
public protocol ConnectionEvent: Event {
    var connectionId: String { get }
}
```
or 
```swift
public protocol ChannelEvent: Event {
    var cid: ChannelId { get }
}
```